src/app/components/navbar.js

```diff
@@ -0,0 +1,17 @@
+const nav = document.createElement('nav');
+const div = document.createElement('div');
+const ul = document.createElement('ul');
+const liContent = ['about', 'menu', 'contact us'];
+
+for (let i = 0; i < liContent.length; i += 1) {
+  const li = document.createElement('li');
+  li.textContent = liContent[i];
+  ul.appendChild(li);
+}
+
+div.classList.add('logo');
+div.textContent = 'Tito';
+
+nav.appendChild(div);
+nav.appendChild(ul);
+document.body.appendChild(nav);
```

Comment:
## Ensure the page is navigable to the rest of the pages.
- [ ] Please add the functionality to ensure that the tabbed menu, when clicked, the content is replaced and the user gets to see what's on the page. Like in this case, they expect to see the `menu` page when they click the menu tab. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/tab_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page)).